src/enumo/workload.rs

```diff
@@ -48,12 +48,26 @@ impl Workload {
         let infile = std::fs::File::open(filename).expect("can't open file");
         let reader = std::io::BufReader::new(infile);
         let mut sexps = vec![];
-        for line in std::io::BufRead::lines(reader) {
-            sexps.push(line.unwrap().parse().unwrap());
+        for line in std::io::BufRead::lines(reader).flatten() {
+            if let Ok(sexp) = line.parse() {
+                sexps.push(sexp);
+            } else {
+                println!("Skipping invalid s-expression: {}", line);
```

Comment:
So we just want to move on to the other expressions and skip invalid ones. We intentionally don't want to throw an exception here, right?